.travis.yml

```diff
@@ -68,3 +80,11 @@ script:
     - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
     - tmp=`mktemp -d`; mkdir $tmp/root $tmp/runroot; sudo PATH="$PATH" ./buildah.test -test.v -root $tmp/root -runroot $tmp/runroot -storage-driver vfs -signature-policy `pwd`/tests/policy.json -registries-conf `pwd`/tests/registries.conf
     - cd tests; travis_wait 60 sudo PATH="$PATH" ./test_runner.sh
+    - cd ..
+    - echo "Run container-diff tests in Fedora container..."
+    - echo "Travis/host environment:"
+    - env
+    - echo
+    - echo "Running tests in SPC using ./hack/run_build.sh"
+    - "./hack/run_build.sh"
+
```

Comment:
Something to consider for a possible follow up.  Pass in the Dockefile file to the script and perhaps you could run the tests multiple times.  Might not be something we'd want to do in Travis CI (thinking further), but might be nice to be able to run multiple dockerfiles through on the command line.

Reply:
not sure what you mean by `run the tests multiple times`. Could you give an example please?

Reply:
or do you mean a single test but with multiple Dockerfiles? And is there a need to use multiple Dockerfiles?

Reply:
Might be nice to do './hack/run_build.sh Dockerfile.nginx Dockerfile.OnBuild Dockerfile.withScript' or something along those lines so you wouldn't be limited to just one Dockerfile, but could potentially test 3 or 4 times.

But again, not something for this review, just something for the back of your mind possible enhancement.

Reply:
I think the script should take a single Dockerfile or none.  If None it uses the builtin.

If Dockerfile is specified then it runs with it.  

Then to do what Tom wants  you could just do

for i in "Dockerfile.nginx Dockerfile.OnBuild Dockerfile.withScript"; do ./hack/run_build.sh $i; done